docs/faq.md

```diff
@@ -0,0 +1,5 @@
+# FAQ
+
+## Newly Registered Users Don't Show Up
```

Comment:
Just so you know, [this](https://github.com/scalableminds/webknossos/pull/2994) PR has been merged and inactive users are now shown above the users list if they registered within the last 2 weeks (see screenshot in PR) :)